docs/Using-Fleet/Automations.md

```diff
@@ -68,21 +68,17 @@ Follow the steps below to configure Jira or Zendesk as a ticket destination:
 6. Select **Enable vulnerability automations** and choose **Ticket**.
 7. Under **Ticket destination**, select your ticket destination and select **Save**.
 
-The Jira and Zendesk ticket destinations are currently in beta.
+The Jira and Zendesk ticket automations are currently in beta.
```

Comment:
@noahtalerman, are the Jira and Zendesk automations still considered beta?

Reply:
> are the Jira and Zendesk automations still considered beta?

@zwass do you know the answer to this?

Tagging Zach because Zach is the DRI for `When a feature is advertised as "beta"`: https://fleetdm.com/handbook/people#directly-responsible-individuals